.gitignore

```diff
@@ -9,3 +9,5 @@ docs/src
 *.swo
 *.swp
 /private/
+*build*
+.idea
```

Comment:
Ide specific folders should be in your global git config, not per project (that includes `.idea`, `.vscode`, etc..)